app/controllers/staff/activities_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class Staff::ActivitiesController < Staff::BaseController
+  include Secured
+
+  def index
+    @activities = Fund.for_user(current_user).collect { |fund| fund.activity }
+  end
+
+  def show
+    @activity = Activity.find(id)
+  end
+
+  def new
+    @activity = Activity.new
+  end
+
+  def create
+    @activity = activity.new(activity_params)
+
+    if @activity.valid?
+      @activity.save
+      flash[:notice] = I18n.t("form.activity.create.success")
+      redirect_to activity_path(@activity)
+    else
+      render :new
+    end
+  end
+
+  private
+
+  # This can/should be made better in future iterations!
```

Comment:
Sounds like it could be reused by other controllers to come. The method itself isn't used in the commit that adds it, was it meant to be left in?

Reply:
Just seen the commit is `WIP`

Reply:
Yeah, I've decided to take it out as it's out of scope. I've left the tests in (skipped) to remind us to do it later!